cmd/scollector/collectors/iterable.go

```diff
@@ -2,68 +2,92 @@ package collectors
 
 import (
 	"context"
-	"fmt"
 	"time"
 
 	"bosun.org/cmd/scollector/conf"
 	"bosun.org/metadata"
 	"bosun.org/opentsdb"
+	"bosun.org/slog"
 	"github.com/bosun-monitor/statusio"
 )
 
+// iterable.io collector
+// to use add this in your scollector.toml:
+// [Iterable]
```

Comment:
Well there is the metadata descriptions etc. Also when I wrote that statusio library I did it by reverse engineering the json api, I couldn't actually find documentation.

We can just keep it as is, I'm worried I just started scope creep.